codegen/smithy-ruby-rails-codegen-test/model/protocol-test/http-headers.smithy

```diff
@@ -109,13 +108,6 @@ apply InputAndOutputWithHeaders @httpRequestTests([
     },
 ])
 
-apply InputAndOutputWithHeaders @skipTests([
```

Comment:
I thought there were two tests for this one, I could be wrong.

Reply:
You're right - I think we just never ported it over.  I'll add it.